qs.gemspec

```diff
@@ -21,9 +21,9 @@ Gem::Specification.new do |gem|
   gem.add_dependency("dat-worker-pool", ["~> 0.4"])
   gem.add_dependency("hella-redis",     ["~> 0.2"])
   gem.add_dependency("ns-options",      ["~> 1.1"])
-  gem.add_dependency("oj",              ["~> 2.11"])
   gem.add_dependency("SystemTimer",     ["~> 1.2"])
 
   gem.add_development_dependency("assert", ["~> 2.12"])
+  gem.add_development_dependency("json",   ["~> 1.8"])
```

Comment:
@jcredding I thought we weren't going to explicitly bring this in?  Just rescue the load error if it can't be loaded and provide a nicer message.  I'd prefer to not add it as a formal dependency (esp since it isn't needed in 1.9+).